src/components/Table.js

```diff
@@ -62,24 +58,38 @@ export default class Table {
       this.activeColumns = Object.keys(this.maptable.columnDetails);
     }
 
-    this.header.selectAll('tr')
+    // make table header fixed
```

Comment:
We could just have an option `options.header.className` that adds the `mt-header-fixed`.

```css
.mt-header-fixed {
  position: sticky;
  top: 0px;
}
```

That could take into consideration larger uses cases than just sticky headers.

Reply:
I updated the table component with new 'className' option. I kept the 'top' option as it is, so that we can give header top space without adding it inside any class, as this value will differ page to page based where maptable is used (for this sticky header usecase). One can just ignore this 'top' option if not needed for any other future use cases.